src/gvpn_controller.py

```diff
@@ -110,7 +117,7 @@ def create_connection_req(self, data):
         self.ondemand_create_connection(uid, send_req=True)
 
     def serve(self):
-        socks, _, _ = select.select(self.sock_list, [], [], CONFIG["wait_time"])
+        socks, _, _ = select.select(self.sock_list, [], [], CONFIG["wait_time"]) # Select sockets ready to be served
```

Comment:
`select` is a bit more complicated than just this. While I think it may be good to make a more detailed comment about how `select` works as it's a fairly high-level networking call, and also about how serve is used in a while loop (see the bottom of the file).